api/v1beta1/disruption_types.go

```diff
@@ -112,8 +112,8 @@ func (dd DisruptionDuration) Duration() time.Duration {
 type DisruptionStatus struct {
 	IsStuckOnRemoval bool `json:"isStuckOnRemoval,omitempty"`
 	IsInjected       bool `json:"isInjected,omitempty"`
-	// +kubebuilder:validation:Enum=NotInjected;PartiallyInjected;Injected
-	// +ddmark:validation:Enum=NotInjected;PartiallyInjected;Injected
+	// +kubebuilder:validation:Enum=NotInjected;PartiallyInjected;Injected;PreviouslyInjected
```

Comment:
Why do we need to change this now? Does it mean we were never updating to this status before in the update status function?
Would it be worth adding a test case [here](https://github.com/DataDog/chaos-controller/blob/4ee14389984f7b421ddf9571955ba5ce7c356060/controllers/disruption_controller_test.go#L256) to be sure that this status is now set?

Reply:
It means we were never updating to this status beforehand, yep. I will add a test case :thubs: